src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -1391,7 +1390,23 @@ pre.rust {
 	border-bottom: 1px solid;
 	margin-bottom: 6px;
 }
-
+#sidebar-toggle > button {
+	background: none;
+	color: inherit;
+	cursor: pointer;
+	text-align: center;
+	border: none;
+	outline: none;
+	position: absolute;
+	top: 0;
+	bottom: 0;
+	left: 0;
+	right: 0;
+	width: 100%;
```

Comment:
I don't think you need to set this because you already have `left: 0` and `right: 0`.

Reply:
Can you add a code comment explaining it please? I'm pretty sure if I wondered about it, someone else might as well in the future.

Reply:
Thanks!